### PR TITLE
Style guide v4

### PR DESCRIPTION
**Pull request by gemfarmer**
Fixes issue(s) #2383 

[CircleCI](https://circleci.com/gh/18F/18f.gsa.gov/tree/sg-4)

[:sunglasses: PREVIEW](https://federalist.fr.cloud.gov/preview/18f/18f.gsa.gov/sg-4/styleguide/)

[Preview README for this branch](https://github.com/18F/18f.gsa.gov/blob/sg-4/README.md)

Changes proposed in this pull request:
- Addresses [this style guide review issue](#2383)

/cc @elainekamlley @jenniferthibault

### CODE REVIEW

**Comment by jenniferthibault** [comment]
Awesome, Brian! I can take a look at this tomorrow. 📆 ☠️ today 😢

File: _styleguide/subpages/images.md [hunk 1]
```diff
@@ -87,7 +121,46 @@ This is a list of all of the SVGs that are available as includes (in the `_inclu
 
 ---
 
-### Image list
+### Adding images
```
Comment (jenniferthibault) [inline _styleguide/subpages/images.md]: This section is in a different order on the page from the left-hand menu

File: _styleguide/subpages/images.md [hunk 2]
```diff
@@ -87,7 +121,46 @@ This is a list of all of the SVGs that are available as includes (in the `_inclu
...
+{% include details-code.html
+   title='svg-catalog'
+   content=source_ref_codeblock
+   lang="markdown"
+   description=source_ref_description
+%}
+
+---
+
+### Image lists
```
Comment (jenniferthibault) [inline _styleguide/subpages/images.md]: I couldn't get this item to link from the side-nav

Comment (jenniferthibault) [inline _styleguide/subpages/images.md]: Same with the last child element in the "Components" section

**Comment by gemfarmer** [comment]
@jenniferthibault I think the bugs you mentioned have been fixed!

**Pull request merged**